diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -375,11 +380,11 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
         if (psi.children.isNotEmpty() && (!psi.isFirstChildElementType(DOT_QUALIFIED_EXPRESSION) &&
                 !psi.isFirstChildElementType(SAFE_ACCESS_EXPRESSION))) {
             val firstChild = psi.firstChild
+            if (firstChild.isFirstChildElementType(DOT_QUALIFIED_EXPRESSION) ||
```

Comment:
Why is this block moved?

Reply:
Because there was incorrect code

Reply:
In what way? Why was it working before? If it wasn't, was in covered in incorrect tests? Have you updated those tests?

Reply:
There was an `if` inside another `if` that could never execute, I changed, and there is a test covering new `if`.